src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -1,5 +1,27 @@
 import React from 'react';
+import { Comment } from '../../Types/Comment';
 
-export const CommentInfo: React.FC = () => (
-  <>Put the comment here</>
+type CommentInfoProps = {
+  comment: Comment,
+};
+
+export const CommentInfo: React.FC<CommentInfoProps> = ({ comment }) => (
+  <div className="CommentInfo">
+    <div className="CommentInfo__title">
+      <strong className="CommentInfo__name">{comment.name}</strong>
```

Comment:
you can destructure comment 
```suggestion
      <strong className="CommentInfo__name">{name}</strong>
```